src/ansys/grantami/recordlists/_connection.py

```diff
@@ -870,20 +873,8 @@ def search_for_audit_log_entries(
         """
 
 
-class _RecordListsApiClient2025R2(RecordListsApiClient):
-    """2025 R2 implementation of the RecordListsApiClient interface."""
-
-    _api = v2025r2api
-    _models = v2025r2models
-
-    def __init__(
-        self,
-        session: requests.Session,
-        service_layer_url: str,
-        configuration: SessionConfiguration,
-    ):
-        logger.debug("Creating RecordListsApiClient for Granta MI 2025 R2")
-        super().__init__(session, service_layer_url, configuration)
+class _RecordListsApiClientWithAuditLog(RecordListsApiClient, ABC):
```

Comment:
I know this is a bit of a departure from what we said we'd do last release, but the only difference between the two is the log message (could be factored out), and the versions of the api and models modules.